penpost/penpost-web
# Gaj/feature/user flow interactions

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

This PR includes additional conditional rendering for UI/UX considerations


Issues closed #36 #37

### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.

Initially, no consideration was put towards the type of information displayed of a user based on if they were or not logged in. We applied state to check if a user is logged in and what type of information is thus displayed or not displayed.  

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

State is held in `App` and passed down to `Header` and `Login` components. This is where the bulk of the changes were made. 

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please check the vercel of this build. Run through logging in and logging out to see the different hamburger menu options provided and different information on the home page. 

### Steps To Take
> Any questions and possible steps moving forward.

Consider adding this state check to whether or not a person can navigate to other routed pages. (If not logged in, the user is redirected to the login page)
